examples/index.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const http = require('http');
+const dispatcher = require('httpdispatcher');
+
+const opentracing = require('opentracing');
+const hawkularAPM = require('../index');
+
+opentracing.initGlobalTracer(new hawkularAPM.APMTracer());
+const span = opentracing.globalTracer().startSpan('printUserInfo');
+
+const SERVER_PORT = 9000;
+
+// Hello handler
+dispatcher.onGet("/hello", function(req, res) {
+    const serverSpan = opentracing.globalTracer().startSpan('hello');
+    res.writeHead(200);
+    res.end('Hello from Node.js!');
+    span.finish();
+});
+
+dispatcher.onGet("/a", function(req, res) {
+    const serverSpan = opentracing.globalTracer().startSpan('a');
+    serverSpan.setTag('span.kind', 'server');
```

Comment:
Same as below - not sure we should rely on the span.kind tag. Instead when receiving a server request, we should use the trace.extract to get the span context, and use childOf - and use this to determine the span represents a Consumer.